sysapi/sysapi/EncryptParam.c

```diff
@@ -35,7 +35,7 @@ TSS2_RC Tss2_Sys_GetEncryptParam(
     const uint8_t **encryptParamBuffer)
 {
     _TSS2_SYS_CONTEXT_BLOB *ctx = syscontext_cast(sysContext);
-    TPM2B *encryptParam;
+    void *offset;
```

Comment:
UINT16 *offset instead of void *

Reply:
I will change to `uint8_t *offset` since it refers to the buffer...